jina/parsers/__init__.py

```diff
@@ -89,6 +89,8 @@ def set_gateway_parser(parser=None):
     mixin_comm_protocol_parser(parser)
     mixin_gateway_parser(parser)
     mixin_pea_parser(parser)
+    mixin_base_pod_parser(parser)
```

Comment:
this mixin base should not be added I think

Reply:
What is the reason it should not be added? It is a gateway pod and would be the first step towards scaling the gateway.

Reply:
changed it since we don't scale the gateway for now